src/voice-pe/using-the-device/new-ha-instance

```diff
@@ -0,0 +1,45 @@
+---
+zendesk:
+  article_id: 30670495779997
+  name: Connecting your Home Assistant Voice PE to a new Home Assistant Instance
+  position: 10
+  labels: voice pe, operation, using the device
+---
+
+When connecting your Home Assistant Voice Preview Edition (VPE) device to a new Home Assistant instance, a factory reset is required. This is because the VPE device stores an encryption key tied to the original Home Assistant instance, even if the device is removed from that setup.
+A factory reset removes this key and allows the device to securely pair with a new instance. This may be necessary if you’ve reinstalled Home Assistant from scratch, are setting up a new environment, or want to use the device with a different system.
+
+## To reset the device
```

Comment:
The topic title is Connecting your VPE to a new instance, but the procedure title is resetting the device. What are we trying to do here, what is the actual procedure?